test/hotspot/jtreg/compiler/loopopts/superword/CoLocatePackMemoryState.java

```diff
@@ -24,7 +24,6 @@
 
 /**
  * @test
- * @requires vm.compiler2.enabled
```

Comment:
Some tests uses this requires when they use C2 specific flags. Like in this case: `LoopMaxUnroll` is C2 flag.
If you want to run such tests with any VM (which may not include C2) you need to add `-XX:+IgnoreUnrecognizedVMOptions`